src-docs/src/views/page/page_example.js

```diff
@@ -80,6 +80,28 @@ export const PageExample = {
       ],
       text: (
         <div>
+          <p>
+            EUI provides a family of components using the{' '}
+            <EuiCode>EuiPage</EuiCode> prefix that work together to build
+            consistant page layouts that work responsively. The below example
+            utilizes the following major components
```

Comment:
Last sentence seems duplicative and we can be more concise without it. 
```suggestion
            consistent page layouts that work responsively. 
```